src/LaunchDarkly.ServerSdk/Files/FileDataSource.cs

```diff
@@ -88,6 +90,10 @@ private void LoadAll()
                     var data = _parser.Parse(content);
                     data.AddToData(allData);
                 }
+                catch (FileNotFoundException) when (_skipMissingPaths)

```

Comment:
To enable the scenario I described yesterday:


Watching current.json and old.json

```
File.Move( "current.json", "old.json" );  // old would get reloaded with same rules
File.Move( "new.json", "current.json" );  // merge of old and new would get reloaded
File.Delete( "old.json" );  // new would get reloaded
```